zulip/integrations/twitter/twitter-bot

```diff
@@ -198,6 +206,16 @@ for status in statuses[::-1][:opts.limit_tweets]:
     text_to_check = [status.text, status.user.screen_name]
     text_to_check.extend(url.expanded_url for url in status.urls)
 
+    # Check that the tweet doesn't contain any terms that
+    # are supposed to be excluded
+    exclude = False
+    for term in excluded_terms:
+        if any(term.lower() in text.lower() for text in text_to_check):
```

Comment:
A minor optimization here would be to lowercase text_to_check strings outside the loop.

Reply:
Will be done, thanks for reviewing